app/controllers/identify_controller.rb

```diff
@@ -65,6 +65,18 @@ def client_cert
 
   def process_cert(raw_cert)
     cert = Certificate.new(OpenSSL::X509::Certificate.new(raw_cert))
+    validation_result = cert.validate_cert(is_leaf: true)
+
+    logger.info({
+      name: 'Certificate Processed',
+      signing_key_id: cert.signing_key_id,
+      key_id: cert.key_id,
+      issuer: cert.issuer.to_s,
+      card_type: cert.card_type,
+      valid_policies: cert.valid_policies?,
+      valid: validation_result == 'valid',
+      error: validation_result == 'valid' ? validation_result : nil,
```

Comment:
invert this, right? because a valid vert should not have an error?
```suggestion
      error: validation_result != 'valid' ? validation_result : nil,
```

Reply:
Yep 🤦🏼‍♂️